ext/ddtrace.c

```diff
@@ -1656,6 +1656,32 @@ PHP_FUNCTION(dd_trace_peek_span_id) {
     RETURN_STR(ddtrace_span_id_as_string(ddtrace_peek_span_id()));
 }
 
+/* {{{ proto void dd_trace_close_all_spans_and_flush() */
+PHP_FUNCTION(dd_trace_close_all_spans_and_flush) {
+    UNUSED(execute_data);
```

Comment:
```suggestion
    zend_parse_parameters_none();
```
We're going to move away from "log_onceerrf" in favour of proper exceptions for invalid code. Let's do it for new code straight away.